home/tests/test_api.py

```diff
@@ -34,7 +34,7 @@ def test_tag_filtering(self):
         # it should return all pages for no tag
         response = self.client.get("/api/v2/pages/")
         content = json.loads(response.content)
-        self.assertEquals(content["count"], 59)
+        self.assertEquals(content["count"], 5)
```

Comment:
@KaitCrawford yes, the test data was 1000+ lines and given how often the export/import has been changing, it is very tedious for me to manually edit the 1000+ lines each time, I do want to make a better and more rigorous content sheet that isnt just the WHO content sheet, it did also occur to me while editing it that it might not be the best to have this active content sitting in a test